modelmapper/loader.py

```diff
@@ -9,54 +10,131 @@ def select(fields):
         raise ImportError('Please install SQLAlchemy')
 
 
-class PostgresBulkLoaderMixin():
+class BaseLoaderMixin():
     """
-    Postgres Specific Loader that backs up raw data into S3.
-    Handles Bulk inserts and ignores when there were errors and continues
+    Base class for loaders. Completely db and data structure agnostic.
+    REQUIRED: Override insert_row_into_db
     """
+    def pre_row_insert(self, row: dict, session, model):
+        """Override to add any logic for rows before being loaded"""
```

Comment:
You mean "after"?

Reply:
No, this method is run with the row before the row is inserted into the db